sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/README.md

```diff
@@ -9,16 +9,14 @@ We need to ensure that this [environment checklist][ready-to-run-checklist] is c
 
 ### Configure web app
 1. Search for and select your tenant in **Azure Active Directory**.
-1. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-protal-manage.png "Protal manage")
+1. Under Manage In the same tenant, select **App registrations** -> **New registration**.![Protal manage](docs/image-portal-manage.png "Protal manage")
 1. The registered application name is filled into `webapp`, select **Accounts in this organizational directory only**, click the **register** button.![Register a web app](docs/image-register-a-web-app.png "Register a web app")
-1. Under **webapp** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later.)![Creat secrets](docs/image-creat-secrets-app.png "Creat secrets")
+1. Under **webapp** application, select **Certificates & secrets** -> **new client secret**, expires select **Never**, click the **add** button.(Remember to save the secrets here and use them later.)![Creat secrets](docs/image-creat-secrets-app.png "Creat secrets")![Creat secrets](docs/secret_value.png "Creat secrets")
 1. Under **webapp** application, select **Authentication** -> **Add a platform**, select **web** platform, redirect urls set to `http://localhost:8080/login/oauth2/code/`, check the **Access Tokens** and **ID Tokens** checkboxes, click **configure** button.![Add a platfform](docs/image-add-a-platfform.png "Add a platfform")
-1. Under **webapp** application, select **API permissions** -> **Add a permission**, select **Microsoft Graph**. Next, search `Directory.AccessAsUser.All` via **select Permissions**, check the check box, click **add permissions** button.(`User.Read` is created automatically, we need to keep it.)![Api permission](docs/image-api-permissions.png "Api permission")
+1. Under **webapp** application, select **API permissions** -> **Add a permission**, select **Microsoft Graph**. Next, search `Directory.Read.All` via **select Permissions**, check the check box, click **add permissions** button.(`User.Read` is created automatically, we need to keep it.)![Api permission](docs/image-api-permissions.png "Api permission")
 1. Similarly, add the following permissions: 
    - **user_impersonation** in **Azure Service Management**,
    - **ActivityFeed.Read**, **ActivityFeed.ReadDlp**, **ServiceHealth.Read** in **Office 365 Management APIs**.![Add permissions](docs/image-add-permissions.png "Add permissions")
-1. click **Grant admin consent for...**.![Grant permission](docs/image-grant-permission.png "Grant permission")
-11. Manually remove the admin consent for **user_impersonation**.(Easy to see incremental authorization.)![Final](docs/image-final.png "Final")
```

Comment:
Do you confirmed that the webapp still works after we delete the step of `admin consent`?

Reply:
I guess it will have problem about `office` client.  Please confirm.

If it have problem about `office` client.
Please use some other client to replace `office`, the new client does not require admin consent.

Reply:
> I guess it will have problem about `office` client. Please confirm.
> 
> If it have problem about `office` client.
> Please use some other client to replace `office`, the new client does not require admin consent.

Yes, there is problem. It looks like other APIs need tenant to have a subscription.